mpi4jax/collective_ops/send.py

```diff
@@ -31,11 +33,13 @@ def mpi_send_impl(x, dest, tag, comm):
     # TODO: make this support gpus (use cupy?)
     inpt = _np.asarray(x)
     comm.Send(inpt, dest=dest, tag=tag)
-    return inpt
+    return 0
```

Comment:
do we need this output? Why 0?

Reply:
No, but custom calls need to return *something*. I thought about returning a size-0 array, but that would make it awkward to introduce data dependencies if you for some reason don't want to use omnistaging. So we could return 0 as a success return code instead.